source/testing/testing-models.md

```diff
@@ -28,26 +28,25 @@ export default Model.extend({
 ```
 
 Now let's create a test which will call `levelUp` on the player when they are
-level 4 to assert that the `levelName` changes. We will use `moduleForModel`:
+level 4 to assert that the `levelName` changes. We will use `module` together with the `setupTest` helper method:
 
 ```tests/unit/models/player-test.js
-import { moduleForModel, test } from 'ember-qunit';
-import { run } from '@ember/runloop';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { run } from "@ember/runloop";
 
-moduleForModel('player', 'Unit | Model | player', {
+module('Unit | Model | player', function(hooks) {
   // Specify the other units that are required for this test.
-  needs: []
-});
-
-test('should increment level when told to', function(assert) {
-  // this.subject aliases the createRecord method on the model
-  const player = this.subject({ level: 4 });
+  test('should increment level when told to', function(assert) {
+    // this.subject aliases the createRecord method on the model
+    const player = run(() => this.owner.lookup('service:store').createRecord('player'));
 
-  // wrap asynchronous call in run loop
-  run(() => player.levelUp());
+    // wrap asynchronous call in run loop
```

Comment:
there is no async/await here so calling this async in the comment makes it very confusing